monorepo/website/src/pages/news/index.astro

```diff
@@ -3,6 +3,13 @@ import MdLayout from '../../layouts/MdLayout.astro';
 
 // Add new news items here
 const newsItems = [
+    {
+        slug: '2024-12-09-expanding-pathoplexus',
```

Comment:
```suggestion
        slug: '2024-12-09-expanding-pathoplexus-mpox',
```

Reply:
out of SEO superstition I'll add mpox, so it shows up higher up when people google mpox